### PR TITLE
Implement basic UI (as close as possible to mock-up for now)

### LINKED ISSUE #49: Initial UI edits

### PR DESCRIPTION
**Pull request by PeterHW963**
fixes #49

### CODE REVIEW

File: src/main/resources/view/MainWindow.fxml [hunk 1]
```diff
@@ -11,10 +11,11 @@
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
 
+<?import javafx.scene.layout.HBox?>
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1"
-         title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+         title="Finding brUdders" minWidth="700" minHeight="600" onCloseRequest="#handleExit">
   <icons>
-    <Image url="@/images/address_book_32.png" />
+    <Image url="@/images/udder_icon2.png" />
```
Comment (XinghuaJulia) [inline src/main/resources/view/MainWindow.fxml:18]:
```suggestion
    <Image url="@/images/udder_icon2.png"/>
```

File: src/main/resources/view/CommandBox.fxml [hunk 2]
```diff
@@ -4,6 +4,6 @@
 <?import javafx.scene.layout.StackPane?>
 
 <StackPane styleClass="stack-pane" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
-  <TextField fx:id="commandTextField" onAction="#handleCommandEntered" promptText="Enter command here..."/>
+  <TextField fx:id="commandTextField" onAction="#handleCommandEntered" promptText="Enter command here..." />
```
Comment (XinghuaJulia) [inline src/main/resources/view/CommandBox.fxml:7]:
```suggestion
  <TextField fx:id="commandTextField" onAction="#handleCommandEntered" promptText="Enter command here..."/>
```

**Review by XinghuaJulia — COMMENTED** [review]
LGTM, minor nits with the CSS formatting. Namely spacing issues.

**Review by psh12320 — COMMENTED** [review]
LGTM!

**Pull request merged**